src/graphics/text.c

```diff
@@ -21,6 +21,19 @@ static struct {
     int y_offset;
 } input_cursor;
 
+static struct {
+    const char * string;
+    int width[FONT_TYPES_MAX];
+} ellipsis = { "...", { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 } };
```

Comment:
You can omit the array of zeroes: when you initialize `string`, the rest of the struct's variables is set to 0.

Reply:
I didn't know about that.

For readability's sake, how about `ellipsis = { "...", { 0 } };` ?

Reply:
Changed as requested, also rebased to prevent many commits. Kept `{ "...", { 0 } }` just to make things clearer.